data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -74,8 +74,12 @@
 ]
 TILES_ROUND_NUM_DECIMALS = 2
 
-# FEMA rounding suffix
-FEMA_SUFFIX = "loss rate"
+# FEMA rounding columns
+FEMA_ROUND_NUM_COLUMNS = [
+    "Expected agricultural loss rate (Natural Hazards Risk Index)",
```

Comment:
Oh, I'd actually vote we reference these as `field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD` instead of as strings.

Reply:
ohhh, yeah that makes more sense. Updated.